ebs/ebs.go

```diff
@@ -589,6 +598,15 @@ func (d *Driver) CreateVolume(req Request) error {
 		if err := fs.FormatDevice(volume.Device, d.DefaultFSType); err != nil {
 			return err
 		}
+		log.Debugf("Launching snapshot creation for brand new volume: %s", volumeName)
+		request := &CreateSnapshotRequest{
+			VolumeID:    volumeID,
+			Description: fmt.Sprintf("Initial snapshot of Convoy volume"),
+			Tags: 		 newTags,
+		}
+		if _, err := d.ebsService.CreateSnapshot(request); err != nil {
+			log.Debugf("Failed to create snapshot for %s: %+v", volumeName, err)
```

Comment:
I see, how would you feel about adding a comment stating as much, and logging any errors as `log.Error` instead of `Debug`?